Content/.template.config/template.json

```diff
@@ -150,7 +181,8 @@
           }
         }
       ]
-    },"**/*.fsx": {
+    },
+    "**/*.fs*": {

```

Comment:
is this change intentional?

Reply:
I can see you also now use `//#if` instead of `#if` in `.fsx` script - is that this change that allowed that? Maybe we can get rid of this entry completely then?

Reply:
@theprash one for you.

Reply:
Right, makes sense. So you commented #if in fsx for better editor support? I think I like it this way now